ShiftLogger.Mefdev/ShiftLoggerUI/Inputs/UserInput.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Globalization;
+using Spectre.Console;
+
+namespace ShiftLogger.Mefdev.ShiftLoggerUI.Inputs;
+
+public class UserInput
+{
+    public UserInput()
+    {
+
+    }
+
+    public string GetId()
```

Comment:
🟠 **Return Type**

💡 You have a method called `GetId`, and you validate that you are getting a number ID back, so why not return an `int` instead of performing the parsing/casting in the calling method?